stimuli_structure/stimuli/stimuli.json

```diff
@@ -0,0 +1,23 @@
+{
+    "stimulus_id": {
+        "Description": "The stimulus ID included in the stimuli directory"
+    },
+    "type": {
+        "Description": "The type of stimulus"
+    },
+    "present": {
```

Comment:
Maybe add that this is a boolean? This seems to work as long as it's clarified. The Google Doc there was an optional URL column however.